intro.md

```diff
@@ -258,45 +290,43 @@ Note: analogy: you can bring your project to Black&Decker, and then you'll have
 <!--s-->
 # Security nightmare
 
-- Need to allow other people on your platform for proximity
-- And they can use whatever software they want on the platform
+- You share the platform with others <!-- .element: class="fragment"-->
+- You can use whatever they want <!-- .element: class="fragment"-->
+- Streams have access control lists <!-- .element: class="fragment"-->
+- Tenants are separated <!-- .element: class="fragment"-->
 
 <!--v-->
 <!-- .slide: data-transition="fade" -->
 ## Base DSH
 ![DC/OS](images/dsh/dsh-tenants.svg)<!-- .element: class="stretch" style="background:none; border:none; box-shadow:none;" width="100%" -->
+Note: in your tenant, you run docker containers. Correct use of docker is _forced_. Docker is already quite secure, but the DSH ensures you can't hack your way out of your container (you can't be root)
```

Comment:
I'd phrase this differently.
Docker containers are meant to isolate processes from the host's environment they run on. This is extremely important when sharing underlying hosts with other processes, possibly owned by other tenants. Therefore, we regulate the use of Docker&mdash;e.g. your process mus run with a specific uid.

Reply:
Yeah, that's better. Altered the text.